CartoDB/cartodb
# Several issues fixed for basemap part

- Added the custom scroll to the add-basemap-modal content (ticket: #11132).
- Removed the re-render of the basemap carousel each time a category is selected.
- Fixed a problem with the NASA date-picker, where a JS error appeared each time that tab was selected in the add-basemap-modal.
- Fixed a problem with a padding in the WMS text input.

![screen shot 2017-01-18 at 10 53 11](https://cloud.githubusercontent.com/assets/132146/22062205/de13570a-dd78-11e6-9467-7e65ce3c5d3c.png)

_*I encourage the reviewer to check those problems in production and check the difference in the proper staging server later._

**Things to test:**
- Check the stuff described in #11132.
- Be sure the add-basemap-modal is working properly for each tab.
- Check if basemap carousel is working properly after selecting a new category and a new custom map.
- Check if WMS/WMTS panel works properly.
